.github/requirements/conda-env-macOS-ARM64

```diff
@@ -8,11 +8,7 @@ pip=22.2.2
 pillow=10.0.1
 pkg-config=0.29.2
 wheel=0.37.1
```

Comment:
Should we move this whole install to pip instead of conda ?

Reply:
I prefer to make number of small changes, but can do it in followups. My goals is to unblock migration to 3.12, as 3.9 is reaching EOL soon